vsutil/__init__.py

```diff
@@ -279,13 +288,11 @@ def depth(clip: vs.VideoNode,
         return clip
 
     should_dither = _should_dither(curr_depth, bitdepth, range_in, range, clip.format.sample_type, sample_type)
-
     dither_type = fallback(dither_type, Dither.ERROR_DIFFUSION if should_dither else Dither.NONE)
 
```

Comment:
Do we need so many blank lines?

Reply:
This commit actually gets rid of that blank line and groups together the dither logic. :thinking: